src/pages/404.tsx

```diff
@@ -0,0 +1,46 @@
+import { getMessages } from '../utils/locale';
+import { GetStaticProps } from 'next';
+import Head from 'next/head';
+import { FormattedMessage as Msg } from 'react-intl';
+import { Link as AdobeLink, Content, Flex, Heading } from '@adobe/react-spectrum';
+
+const options = {
+    localeScope: [
+        'pages.404',
+        'misc.publicHeader',
+    ],
+};
```

Comment:
This structure was "designed" as a way of passing options to `scaffold()`. Since it's only hard-coded in this use case, we don't need the entire structure. The array is all we need, and since it's only used in one place (line 16) it could be inlined there.